field/generator/internal/templates/element/ops_purego.go

```diff
@@ -6,7 +6,7 @@ const OpsNoAsm = `
 import "math/bits"
 {{- end}}
 
-{{ $mulConsts := list 3 5 13 }}
+{{ $mulConsts := list 3 5 11 13 }}
```

Comment:
I don't think you need the `MulBy11` . i.e. no perf gain, just define a value at 11 in the extension for small fields and mul by that ?